### PR TITLE
Captures errors in Raven

### PR DESCRIPTION
**Pull request by tonytlwu**

### CODE REVIEW

**Comment by seromenho** [comment]
Did you test it?
Maybe `typeof Raven !== 'undefined'`

**Comment by tonytlwu** [comment]
I think it works. It's how it's used in `fliplet-ios-version-2` https://github.com/Fliplet/fliplet-ios-version-2/blob/master/www/js/native.js#L118-L120

**Comment by tonytlwu** [comment]
And here https://github.com/Fliplet/fliplet-api/blob/master/public/assets/fliplet-native/1.0/js/native.js#L116-L118

**Comment by seromenho** [comment]
I'm not confident it will work. You can give it a try on the console to confirm.

File: js/build.js [hunk 1]
```diff
@@ -33,6 +33,13 @@ Fliplet.Widget.instance('image', function (data) {
     $placeholder.fadeInImg(this);
   }).on('error', function(){
     $placeholder.fadeInImg(this);
+
+    if (Raven) {
+      Raven.captureMessage('Error loading image', {
+        user: Fliplet.User.get('id'),
+        image: data.image
```
Comment (seromenho) [inline js/build.js]: We could get more help if you also capture app/page id.
Fliplet.Env.get('appId')
Fliplet.Env.get('pageId')

File: js/build.js [hunk 2]
```diff
@@ -34,9 +34,11 @@ Fliplet.Widget.instance('image', function (data) {
   }).on('error', function(){
     $placeholder.fadeInImg(this);
 
-    if (Raven) {
+    if (typeof Raven !== 'undefined' && Raven.captureMessage) {
       Raven.captureMessage('Error loading image', {
         user: Fliplet.User.get('id'),
+        app: Fliplet.Env.get('appId'),
```
Comment (seromenho) [inline js/build.js]: just naming but why not appId/pageId like we usually set?

Comment (tonytlwu) [reply js/build.js]: Dunno. I do sometimes just use `app`.

Comment (squallstar) [reply js/build.js]: we should definitely have a JS API to handle this...

```
Fliplet.Errors.capture
```

**Pull request merged**